src/Controls/src/Build.Tasks/Controls.Build.Tasks.csproj

```diff
@@ -20,6 +20,7 @@
 	</ItemGroup>
 
 	<ItemGroup>
+		<ProjectReference Include="..\..\Maps\src\Controls.Maps.csproj" />
```

Comment:
The build tasks can't depend on maps because maps is an optional, separate nuget package.

Reply:
Fixed